### DISCUSSION TITLE
blog/a-brudge-between-gap-khmer-dev-to-khmer-lang

### DISCUSSION THREAD

**Discussion by giscus** [vicheanath/vicheablog#17] [General]
# blog/a-brudge-between-gap-khmer-dev-to-khmer-lang

Currently, Cambodia developers that develop tech products really hard to put the Khmer language into their platform because Some developer doesn’t have a big amount of knowledge about the Khmer language, Yes we are Cambodia people but to know every word and every sentence in Khmer include some Baley language

https://vicheablog-c3691fy9a-vicheanath.vercel.app/blog/a-brudge-between-gap-khmer-dev-to-khmer-lang

**Comment by vicheanath** [comment]
Please share your thoughts here.
